util/getNodeJsChangelog.ts

```diff
@@ -1,23 +1,31 @@
 import semVer from 'semver';
 
-export const getNodejsChangelog = (version: string) => {
-  const changelogs = 'https://github.com/nodejs/node/blob/main/doc/changelogs';
+/**
+ * Returns the URL of the Node.js changelog for the specified version.
+ *
+ * @param version The version of Node.js to get the changelog for.
+ * @returns The URL of the Node.js changelog for the specified version.
+ */
+export const getNodejsChangelog = (version: string): string => {
+  const changelogsUrl =
```

Comment:
Any reason for the line break?

Reply:
make's it easier to read and understand

Reply:
I don't think that makes it easier to understand. Could you please revert this change?

Reply:
> Any reason for the line break?

If not do that, the Lint cannot pass :(

Plz see : https://github.com/nodejs/nodejs.org/actions/runs/4689785134/jobs/8320031623

Reply:
I don't think this has anything related to the linting. Linting passes either way. With or without line break.

Reply:
> I don't think this has anything related to the linting. Linting passes either way. With or without line break.

You can have a try and you'll find if you run "prettier:fix", the formation will change like this.

Reply:
> I don't think this has anything related to the linting. Linting passes either way. With or without line break.

Oh I know why……Bcoz I directly run 'npm run prettier:fix' to auto fix the problem. And For the 1st error foramtion PR, it's because the lack of the end new line...

Reply:
No no, the line break is happening because the variable is now `changelogsUrl` (so it is long enough to reach the max line char limit).